scipy/stats/_continuous_distns.py

```diff
@@ -4609,6 +4609,47 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_cauchy_gen(rv_continuous):
+    """A skewed Cauchy random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The pdf is::
+
+        skewcauchy.pdf(x, a) = 1 / (pi * (x**2 / (a * sign(x) + 1)**2 + 1))
+
+    `skewcauchy` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to the usual Cauchy distribution.
+
+    %(after_notes)s
+
+    %(example)s
+    """
+
+    def _argcheck(self, a):
+        return np.isfinite(a) and np.abs(a) != 1
```

Comment:
https://en.m.wikipedia.org/wiki/Skewed_generalized_t_distribution

states that a (called lambda in that wiki page) is restricted to -1 < a < 1

Reply:
Done.